src/main/java/no/nav/pto/veilarbportefolje/domene/Sorteringsfelt.java

```diff
@@ -0,0 +1,119 @@
+    STARTDATO_FOR_AVTALT_AKTIVITET("aktivitet_start"),
+    NESTE_STARTDATO_FOR_AVTALT_AKTIVITET("neste_aktivitet_start"),
+    FORRIGE_DATO_FOR_AVTALT_AKTIVITET("forrige_aktivitet_start"),
+
+    AAP_RETTIGHETSPERIODE("aaprettighetsperiode"),
+
+    MOTER_MED_NAV_IDAG("moterMedNAVIdag"),
+    MOTESTATUS("motestatus"),
+
+    GJELDENDE_VEDTAK_14A_INNSATSGRUPPE("gjeldende_vedtak_14a_innsatsgruppe"),
+    GJELDENDE_VEDTAK_14A_HOVEDMAL("gjeldende_vedtak_14a_hovedmal"),
+    GJELDENDE_VEDTAK_14A_VEDTAKSDATO("gjeldende_vedtak_14a_vedtaksdato"),
+
+    UTKAST_14A_STATUS("utkast_14a_status"),
+    UTKAST_14A_STATUS_ENDRET("utkast_14a_status_endret"),
+    UTKAST_14A_ANSVARLIG_VEILEDER("utkast_14a_ansvarlig_veileder"),
+
+    ARBEIDSLISTE_FRIST("arbeidslistefrist"),
+    ARBEIDSLISTE_KATEGORI("arbeidslistekategori"),
+    ARBEIDSLISTE_OVERSKRIFT("arbeidsliste_overskrift"),
+
+    SISTE_ENDRING_DATO("siste_endring_tidspunkt"),
+
+    FODELAND("fodeland"),
+    STATSBORGERSKAP("statsborgerskap"),
+    STATSBORGERSKAP_GYLDIG_FRA("statsborgerskap_gyldig_fra"),
+
+    TOLKESPRAK("tolkespraak"),
+    TOLKEBEHOV_SIST_OPPDATERT("tolkebehov_sistoppdatert"),
+
+    BOSTED_KOMMUNE("kommunenummer"),
+    BOSTED_BYDEL("bydelsnummer"),
+    BOSTED_SIST_OPPDATERT("bostedSistOppdatert"),
+
+    CV_SVARFRIST("neste_svarfrist_stilling_fra_nav"),
+
+    ENSLIGE_FORSORGERE_UTLOP_YTELSE("enslige_forsorgere_utlop_ytelse"),
+    ENSLIGE_FORSORGERE_VEDTAKSPERIODETYPE("enslige_forsorgere_vedtaksperiodetype"),
+    ENSLIGE_FORSORGERE_AKTIVITETSPLIKT("enslige_forsorgere_aktivitetsplikt"),
+    ENSLIGE_FORSORGERE_OM_BARNET("enslige_forsorgere_om_barnet"),
+
+    BARN_UNDER_18_AR("barn_under_18_aar"),
+
+    BRUKERS_SITUASJON_SIST_ENDRET("brukersSituasjonSistEndret"),
+    UTDANNING_OG_SITUASJON_SIST_ENDRET("utdanningOgSituasjonSistEndret"),
+
+    HUSKELAPP_KOMMENTAR("huskelapp_kommentar"),
+    HUSKELAPP_FRIST("huskelapp_frist"),
+    HUSKELAPP("huskelapp"),
+
+    FARGEKATEGORI("fargekategori"),
+
+    TILTAKSHENDELSE_TEKST("tiltakshendelse_tekst"),
+    TILTAKSHENDELSE_DATO_OPPRETTET("tiltakshendelse_dato_opprettet");
+
+    /** Filterverdien som vert sendt mellom frontend og backend */
+    public final String value;
+
+    Sorteringsfelt(String value) {
+        this.value = value;
+    }
+
+    /** toString() returnerer verdien til enumen, ikkje namnet.
+     * Eksempel: BOSTED_KOMMUNE.toString() -> "kommunenummer"
+     * "value" er verdien som vert sendt mellom frontend og backend. */
+    @Override
+    public String toString() {
+        return this.value;
+    }
+
+    public static Sorteringsfelt nameFromValue(String value) {
+        for (Sorteringsfelt sorteringsfelt : values()) {
+            if (sorteringsfelt.value.equals(value)) {
+                return sorteringsfelt;
+            }
+        }
+        throw new IllegalArgumentException("Ugyldig verdi for enum: " + value ); // TODO betre feilhandtering ved ugyldige values?
```

Comment:
Feilhandteringen trur eg eg i så fall ville gjort lengst mogleg opp i kallkjeden, for eksempel i controlleren i forbindelse med validering (#valider _så_tidleg_som_mogleg). Med andre ord vi bør ha ein gyldig `Sorteringsfelt` å jobbe med så tidleg som mogleg, i staden for å sende rundt strings.